test/cypress/integration/widgets/agents_spec.js

```diff
@@ -38,14 +38,18 @@ describe('Agents widget', () => {
     });
 
     it('should display agents that match the search phrase', () => {
+        const checkCell = ($tr, number, value) => {
+            cy.wrap($tr)
+                .find(`td:nth-child(${number})`)
+                .then($td => expect($td.text()).to.equal(value));
```

Comment:
Probably not 100% semantically same, but I think equally good and simpler:
```suggestion
                .find(`td:nth-child(${number})`)
                .contains(value);
```

Reply:
After the agreement, equal will stay